### PR TITLE
fuzzdb: Updating the raft lists

### PR DESCRIPTION
**Pull request by kurobeats**
Updating the raft lists as updates have been made by the SecLists project in the last few years

### CODE REVIEW

**Comment by kingthorin** [comment]
Note this is add-on is specifically based on [fuzzDB](https://github.com/fuzzdb-project/fuzzdb). If lists are coming from other sources that might cause a sync/maintenance issues.

**Comment by kurobeats** [comment]
> Note this is add-on is specifically based on [fuzzDB](https://github.com/fuzzdb-project/fuzzdb). If lists are coming from other sources that might cause a sync/maintenance issues.
> 
> Assuming this goes ahead the DCO will need to be addressed and the CHANGELOG will need an entry.

I'm not overly worried about fuzzDB being the source, these particular files haven't been touched by that project in 7 years. On the DCO issue, can I fix that without trashing my existing merge? Happy to update the CHANGELOG, ideally what would you like to see added?

**Comment by thc202** [comment]
Better to also PR the update in fuzzDB project.

**Comment by kingthorin** [comment]
The DCO bit is simple:
```
git commit --amend --no-edit --signoff
git push --force
```

**Comment by kingthorin** [comment]
For the CHANGELOG you can add to the unreleased change block, something like `Updated RAFT lists based on more recent  SecLists contributions`.

File: addOns/fuzzdb/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,9 @@ All notable changes to this add-on will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
+## [9] - 2022-03-18
+- Updated RAFT lists based on more recent SecLists contributions
```
Comment (kingthorin) [inline addOns/fuzzdb/CHANGELOG.md:8]: This should be in the `### Changed` section just below.

Comment (thc202) [reply addOns/fuzzdb/CHANGELOG.md:8]: And the previous line (i.e. `## [9]…`) removed.

Comment (kurobeats) [reply addOns/fuzzdb/CHANGELOG.md]: updated!

**Comment by thc202** [comment]
Could you fix up the commits?

**Comment by kingthorin** [comment]
Just to keep everyone updated. I have been in-touch with the FuzzDB project owner. He needs to regain access to his GitHub account but is working on addressing the upstream PR and getting us access.

**Comment by kurobeats** [comment]
> Could you fix up the commits?

Apologies, what have I missed with the commit?

**Comment by kingthorin** [comment]
Please do an interactive rebase and fix-up or squash into a single commit:

- https://www.kernel.org/pub/software/scm/git/docs/git-rebase.html#_interactive_mode
- https://www.instructables.com/id/How-to-Rebase-Interactive-With-Eclipse-eGit/
- https://wiki.eclipse.org/EGit/User_Guide#Interactive_Rebase

**Comment by kurobeats** [comment]
> Please do an interactive rebase and fix-up or squash into a single commit:
> 
> * https://www.kernel.org/pub/software/scm/git/docs/git-rebase.html#_interactive_mode
> * https://www.instructables.com/id/How-to-Rebase-Interactive-With-Eclipse-eGit/
> * https://wiki.eclipse.org/EGit/User_Guide#Interactive_Rebase

Hopefully I've done it correctly.

**Comment by kingthorin** [comment]
Looks like it 🙂

**Comment by thc202** [comment]
Thank you!

**Review by psiinon — APPROVED** [review]

**Pull request merged**